build/mac/sign_app.sh

```diff
@@ -36,16 +38,23 @@ function check_exit() {
 
 trap check_exit EXIT
 
-# Copy signing script to the packaging directory
-SCRIPT_DIR=$(dirname ${0})
-cp -f "${SCRIPT_DIR}/sign_brave.py" "${PKG_DIR}"
+# brave/scripts/signing_helper.py will retrieve this value when called from
+# sign_chrome.py
+export MAC_PROVISIONING_PROFILE
+
+# Clear output directory. It seems GN auto-creates directory path to the
+# expected outputs. However, the signing script doesn't expect the path to
+# have been created and fails trying to create it again.
+echo "Cleaning $DEST_DIR ..."
+rm -rf $DEST_DIR/*
 
 # Invoke python script to do the signing.
+PARAMS="--input $SOURCE_DIR --output $DEST_DIR --keychain $MAC_SIGNING_KEYCHAIN --identity $MAC_SIGNING_IDENTIFIER --no-dmg --no-notarize"
```

Comment:
@mbacchi, chrome/installer/mac/sign_chrome.py now takes --notarize/--no-notarize, --notary-user, and --notary-password arguments. Possibly, we can use their script for notarization? We aren't creating dmg in this step, so it would have to be notarized separately.

Reply:
That's the approach I'm taking right now, I'm trying to leverage their scripts as much as possible. I'm getting close to something that I can get your opinion on.